packages/react/src/feature-link-list/docs/overview.mdx

```diff
@@ -53,6 +62,87 @@ The feature link list component can be used without a description.
 			label: 'Accept an invite',
 			href: '#',
 		},
+		{
+			label: 'About the department',
+			href: 'https://www.agriculture.gov.au/',
+			target: '_blank',
+		},
 	]}
 />
 ```
+
+## Hierarchy
```

Comment:
I feel this section raises more questions than it answers:

1. Should we have the same `Hierarchy` section in the cards docs?
2. Can you use the simpler link list instead? Or others from the "Related components" list?
3. Will consumers copy this code and have out of context heading levels? Most of the time the premise of the docs examples is code isolation. With headings, we need to respect the wider page.
4. What problem does this pattern solve? Part lorem ipsum, part business related content makes inference hard and there's no 'why/'why not' in the explanation.

If this legitimately solves a problem, then it belongs in the pattern section of the docs, IMO. Also `Information hierarchy` or `Using with other lists of links` would be a better title than `Hierarchy`.